wurst/network/Metadata.wurst

```diff
@@ -0,0 +1,65 @@
+package Metadata
+import GamecacheKeys
+
+public tuple metadata(int intCount, int realCount, int booleanCount, int asciiIntCount, int syncCount)
+
+public function metadata.setIntCount(int value) returns metadata
+	return metadata(value, this.realCount, this.booleanCount, this.asciiIntCount, this.syncCount)
+
+public function metadata.setRealCount(int value) returns metadata
+	return metadata(this.intCount, value, this.booleanCount, this.asciiIntCount, this.syncCount)
+
+public function metadata.setBooleanCount(int value) returns metadata
+	return metadata(this.intCount, this.realCount, value, this.asciiIntCount, this.syncCount)
+
+public function metadata.setAsciiIntCount(int value) returns metadata
+	return metadata(this.intCount, this.realCount, this.booleanCount, value, this.syncCount)
+
+public function metadata.setSyncCount(int value) returns metadata
+	return metadata(this.intCount, this.realCount, this.booleanCount, this.asciiIntCount, value)
+
+/**
+	Simple wrapper class for metadata.
```

Comment:
What exactly is metadata?

Reply:
Updated hotdoc to reflect class purpose better